aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java

```diff
@@ -251,6 +256,9 @@ public void run(String... args) {
                 if (outputFile != null) {
                     Files.deleteIfExists(outputFile);
                 }
+                if (downloadDirectory != null) {
+                    FileUtils.deleteDirectory(downloadDirectory.toFile());
```

Comment:
As we just discussed, should delete the contents rather than the directory since it's mounted from elsewhere. Also, to be extra safe we might want to create a subfolder to put our files in and only delete from that (in case the contents are shared with something else for what ever reason).